tests/integration.js

```diff
@@ -9,6 +9,8 @@ require('../lib/impress');
 
 ncp.limit = 16;
 
+//let timer = setTimeout(taskExit, 5000);
```

Comment:
I guess the reason of integration test flakiness is not whether `api.common` is a part of this repository or a separate NPM package, so we should land this PR regardless of this bug (since it's not related to the commit) and fix it as a separate PR.